x-pack/plugins/cloud_integrations/cloud_links/public/maybe_add_cloud_links/theme_darkmode_hook.ts

```diff
@@ -37,14 +40,23 @@ export const useThemeDarkmodeToggle = ({ uiSettingsClient }: Deps) => {
     },
   });
 
-  const { userSettings: { darkMode: colorScheme } = { darkMode: undefined } } =
-    userProfileData ?? {};
+  const {
+    userSettings: {
+      darkMode: colorScheme = uiSettingsClient.get('theme:darkMode') === true ? 'dark' : 'light',
```

Comment:
`uiSettingsClient` should not be used to check theme;  Core ThemeService would be better, but @clintandrewhall might know best

Reply:
That was already there, I just refactored the file

Reply:
Ah, my mistake - I see it down below now.